scripts/generate-autodocs-registry.js

```
@@ -21,8 +21,13 @@ async function generate() {
   const autodocsRegistry = {};
   const gatherAllPromises = [];
   Registry.default.metadata.components.forEach(async comp => {
+    const path = comp.exportInfo.path;
+    if (!path) {
+      // eslint-disable-next-line no-console
+      console.error('Export info must contain a path to the component.');
```

I think we can throw an error and abort.